docs/changelog.rst

```diff
@@ -2,6 +2,9 @@
 Changelog
 =========
 
+* :bug:`-` Transfers between tracked accounts will now have a correct label in the UI.
+
```

Comment:
I would mention that this is in the case of ETH, otherwise is confusing at it seems that it was fixed before and now again